unixfs/io/dagmodifier_test.go

```diff
@@ -38,7 +39,7 @@ func getNode(t *testing.T, dserv mdag.DAGService, size int64) ([]byte, *mdag.Nod
 		t.Fatal(err)
 	}
 
-	dr, err := NewDagReader(node, dserv)
+	dr, err := NewDagReader(context.TODO(), node, dserv)
```

Comment:
use `context.TODO()` when it's mean to (+ will be) be wired up to something else, and `context.Background()` when it's not.